.travis.yml

```
@@ -1,5 +1,3 @@
-dist: xenial
-sudo: required
 language: node_js
```

```suggestion
os: linux
language: node_js
```

we should probably specify what the default os is for all the other jobs.